### PR TITLE
Updated a few functions to remove compilation errors and added a sample GoogleService-Info.plist file

### PR DESCRIPTION
**Pull request by legalimpurity**
- ‘FIRFunctionsErrorCode' has been renamed to 'FunctionsErrorCode'
- No need to use SetOptions.merge(), setData function arguments changed. Rather than writing options: SetOptions.merge() just write merge: true
- Added a GoogleService-Info.plist tried to modify the configuration from the values in the android sample. With some help might get this project up and running.

### CODE REVIEW

File: Source/Platform/Services/FeedbackService.swift [hunk 1]
```diff
@@ -79,7 +79,7 @@ public final class FeedbackService {
     firebaseFunctions.httpsCallable("sendFeedback").call(feedback) { (_, error) in
       if let error = error as NSError? {
         if error.domain == FunctionsErrorDomain {
-          let code = FIRFunctionsErrorCode(rawValue: error.code) ?? .unknown
+            let code = FunctionsErrorCode(rawValue: error.code) ?? .unknown
```
Comment (morganchen12) [inline Source/Platform/Services/FeedbackService.swift]: Please fix this indent.

File: Source/IOsched/Configuration/GoogleService-Info.plist [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (morganchen12) [inline Source/IOsched/Configuration/GoogleService-Info.plist]: Please revert this file.

File: Source/IOsched/AppDelegate.swift [hunk 3]
```diff
@@ -41,7 +41,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // Firebase setup
     FirebaseApp.configure()
     // Uncomment this line to enable debug logging from Firebase.
-    // FirebaseConfiguration.shared.setLoggerLevel(.debug)
+     FirebaseConfiguration.shared.setLoggerLevel(.debug)
```
Comment (morganchen12) [inline Source/IOsched/AppDelegate.swift]: Please re-comment this code.

**Review by morganchen12 — COMMENTED** [review]
Looks good, had a few nits.

Thanks for doing this!

**Comment by legalimpurity** [comment]
I made a new commit to my fork. Do you want me to create a new PR or is this fine?

**Review by morganchen12 — APPROVED** [review]

**Pull request merged**